src/test/predict_validation_test.cpp

```diff
@@ -39,6 +39,7 @@ class TfsPredictValidation : public ::testing::Test {
     tensorflow::serving::PredictRequest request;
     ovms::ModelConfig modelConfig{"model_name", "model_path"};
     ovms::tensor_map_t servableInputs;
+    ovms::tensor_map_t servableOutputs = ovms::tensor_map_t({});  // not used for TFS
```

Comment:
If it is not used it may be better to use some random tensor info to detect with this test that by accident it will start to be used? Same for KFS